lib/users/unsubscribeUser.js

```diff
@@ -0,0 +1,35 @@
+import { Db } from "mongodb"; //lgtm [js/unused-local-variable]
+
+/**
+ * A function used to unsubscribe a user
+ * @param client {Db} - the connection object for the database
+ * @param cuid {string} - the cuid for the user
+ * @returns {Promise<Collection.insertWriteOpResult>}
+ */
+export async function unsubscribeUser(client, cuid) {
+  return await client.collection("users").findOneAndUpdate(
```

Comment:
Should deleteOne be used here instead. Couldn't help myself from taking a peek

Reply:
I initially thought so too, but we now want to keep the cuid while removing any other data. Reasons can be found in [Design Doc 008](https://github.com/DTS-STN/Alpha-Site/wiki/Design-Doc-008-Unsubscribe).